addons/pos_loyalty/static/tests/tours/PosLoyaltyTour.js

```diff
@@ -255,56 +255,54 @@ registry.category("web_tour.tours").add("PosLoyaltyTour7", {
         ].flat(),
 });
 
-registry.category("web_tour.tours").add('PosLoyaltyTour8', {
+registry.category("web_tour.tours").add("PosLoyaltyTour8", {
     test: true,
-    url: '/pos/web',
+    url: "/pos/web",
```

Comment:
in all modified tour files, removed `url: "/pos/web"` or `url: "/pos/ui"`. There is no need for it since when calling `start_tour`, we already specify the url.